src/reduxTypes.js

```diff
@@ -166,6 +166,9 @@ export type MigrationsState = {|
 
 export type MuteState = MuteTuple[];
 
+/** A map from user IDs to the Unix timestamp in seconds when they were muted. */
+export type MutedUsersState = Immutable.Map<UserId, number>;
```

Comment:
I'm glad to have another case of `Immutable.Map`s with numeric keys! 🎉 b54d68d9c should ensure that these get "revived" from storage with their keys as numbers, instead of strings. The data is persisted as JSON so we've had to be careful about that.